continuous_integration/recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ build:
 
 requirements:
   build:
+    - {{ compiler('c') }} =11
```

Comment:
I set this pin somewhat arbitrarily, but it seems like GCC 11 successfully built the shared object locally for me

Reply:
Would do this by adding a `conda_build_config.yaml` file with `c_compiler_version` specified (like [so]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/6debcd5a454e7163cd1adeae066906617feb6622/recipe/conda_build_config.yaml#L6-L7 ))